ansible-ipi-install/roles/node-prep/tasks/30_req_packages.yml

```diff
@@ -1,8 +1,18 @@
 ---
-- name: Install required packages
-  package:
-    name: "{{ item }}"
-    state: present
-  with_items: "{{ package_list }}"
-  become: yes
+- name: Install packages
+  block:
+  - name: Install required packages
+    package:
+      name: "{{ item }}"
+      state: present
+    with_items: "{{ package_list }}"
+    become: yes
+
+  - name: Install required packages when cache_enabled
+    package:
+      name: "{{ item }}"
+      state: present
+    with_items: "podman"
+    become: yes
+    when: cache_enabled == True
   tags: packages
```

Comment:
nit i know --- but since you decided to put in a block, do we want to remove the become: yes from each task and just have it at the end of the block?